doc/source/create-k8s-cluster.rst

```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
 
 2. Enable the `Kubernetes Engine API <https://console.cloud.google.com/apis/api/container.googleapis.com/overview>`_.
 
-3. Install and initialize the **gcloud command-line tools**. These tools send
-   commands to Google Cloud and lets you do things like create and delete
-   clusters.
+3. Prepare your command line interface. You have two options, using Google's
+   interactive CLI, or working locally from your own shell interface. Instructions
+   for each are detailed below:
 
-   - Go to the `gcloud downloads page <https://cloud.google.com/sdk/downloads>`_
-     to **download and install the gcloud SDK**.
-   - See the `gcloud documentation <https://cloud.google.com/sdk/>`_ for
-     more information on the gcloud SDK.
-   - Install ``kubectl``, which is a tool for controlling kubernetes. From
-     the terminal, enter:
+   * **Google Interactive CLI**. Start the Google Cloud interactive command line
```

Comment:
**Use the Google Cloud Shell**. Start Google Cloud Shell by clicking ...

https://cloud.google.com/shell/docs/